server/src/main/java/org/apache/druid/server/initialization/jetty/JettyRequestLog.java

```diff
@@ -35,11 +35,12 @@ public void log(Request request, Response response)
   {
     if (logger.isDebugEnabled()) {
       logger.debug(
-          "%s %s %s %s",
+          "%s %s %s %s %s",
```

Comment:
the `getStatus` returns type of int, the format specifier here should be `%d`

Reply:
Hi @FrankChen021 , is there a way to suppress the coverage failure? 🤷🏼 
I have tried to deliver a silly unit test to avoid the lack of code coverage error but I can't allocate too much time for it atm.
Any help is deeply appreciated 🙏🏼